### PR TITLE
Bridge Crew edit interface

### LINKED ISSUE #64: Add UI to create/edit a bridge crew
Needs:

* Input fields for member IDs for all bridge crew members
* Start date (9:00 PM on Changing Of The Guard date)
* End date (7:00 PM on Changing Of The Guard date)

**Note: instructions for adding a new bridge crew should be added to the [How To Enter As A Bridge Crew wave](https://rizzoma.com/topic/f8cfddb4811ff4324cebac33742a2b8b) for the incoming engineer to do (ideally with help from the outgoing engineer.**

### PR DESCRIPTION
**Pull request by ZMYaro**
Closes #64.

### CODE REVIEW

**Review by noblelady — APPROVED** [review]
Looks good

File: bridgecrew.py [hunk 1]
```diff
@@ -32,12 +35,74 @@ def get(self):
...
+			else:
+				self.error(404)
+				return
+		
+		template = JINJA_ENVIRONMENT.get_template('bridge_crew_edit.html')
+		self.response.write(template.render(template_vals))
+	
+	def post(self, args):
+		import logging
+		logging.debug('SNTHOISNTHOISNT')
```
Comment (dexw25) [inline bridgecrew.py]: looks like a leftover debug statement

File: bridgecrew.py [hunk 2]
```diff
@@ -32,12 +35,74 @@ def get(self):
...
+		if not require_admin(self):
+			return
+		
+		logging.debug('UPDATE RECEIVED!')
```
Comment (dexw25) [inline bridgecrew.py]: Looks like a leftover debug statement

**Review by dexw25 — CHANGES_REQUESTED** [review]
Looks good except for a couple of extra debug statements

**Comment by ZMYaro** [comment]
Fixed.  Thanks for catching that, @derekw023!

**Review by dexw25 — APPROVED** [review]
Should be good to go now

**Pull request merged**